keepalived/check/check_misc.c

```diff
@@ -71,6 +71,26 @@ dump_misc_check(void *data)
 	log_message(LOG_INFO, "   insecure = %s", misck_checker->insecure ? "Yes" : "No");
 }
 
+static int
```

Comment:
I prefer type bool, return true for same (currently 0), and false if different and change goto err to return false